packages/flutter/lib/src/cupertino/text_theme.dart

```diff
@@ -176,6 +187,10 @@ class CupertinoTextThemeData extends Diagnosticable {
   /// The [TextStyle] of unselected tabs.
   TextStyle get tabLabelTextStyle => _tabLabelTextStyle ?? _defaults.tabLabelTextStyle;
 
+  final TextStyle _tabWideLabelTextStyle;
+  /// Typography of unselected tabs.
+  TextStyle get tabWideLabelTextStyle => _tabWideLabelTextStyle ?? _kDefaultTabWideLabelTextStyle;
```

Comment:
The implementation of `_defaults` should probably be changed accordingly since a new textstyle is added.